src/app/components/Image/index.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import styled from 'styled-components';
-import { filterForBlockType } from '../../helpers/blockHandlers';
+import filterForBlockType from '../../helpers/blockHandlers';
```

Comment:
Should this still be `{ filterForBlockType }`?

Reply:
Ahhh actually with your changes this is now the only exported method in that class. Probably still worth the deconstruction unless the linters complain